browser-interface/packages/shared/session/sagas.ts

```diff
@@ -321,7 +321,9 @@ async function createAuthIdentity(requestManager: RequestManager, isGuest: boole
 
   const ssoIdentity = await SingleSignOn.getIdentity(address)
 
-  if (!ssoIdentity) {
+  const expiredIdentity = ssoIdentity ? isSessionExpired({ identity: ssoIdentity }) : true
+
+  if (!ssoIdentity || expiredIdentity) {
```

Comment:
```suggestion
  if (!ssoIdentity || isSessionExpired({ identity: ssoIdentity })) {
```
Maybe this is simpler